advanced/kernel-output/README.md

```diff
@@ -214,20 +215,44 @@ async function createPanel(): Promise<ExamplePanel> {
 
 ## Make it Run
 
-This example assumes you have a variable, named `df`, in your python kernel that could
-contain a [pandas](https://pandas.pydata.org/) dataframe. You can display it in your panel by adding the following command:
+To make the output area useful, you will request the user to enter a statement
+to be executed by the kernel. Then you will send it to your panel for execution
+and display:
 
 ```ts
-// src/index.ts#L73-L82
+// src/index.ts#L73-L93
 
 commands.addCommand(CommandIDs.execute, {
-  label: 'kernel-output: Show Dataframe',
-  caption: 'Show Dataframe',
+  label: 'kernel-output: Execute Code',
+  caption: 'Execute Code',
   execute: async () => {
+    // Create the panel if it does not exist
     if (!panel) {
       await createPanel();
     }
-    panel.execute('df');
+    // Ask the user about the statement to be executed
```

Comment:
Would "prompt" be more adequate here? Something like "Prompt the user for the statement to execute".